db/migrate/20210115124508_template_kind_registration.rb

```diff
@@ -0,0 +1,16 @@
+class TemplateKindRegistration < ActiveRecord::Migration[6.0]
+  def up
+    template_kind = TemplateKind.unscoped.find_or_create_by(name: 'global_registration')
+    templates = ProvisioningTemplate.unscoped.where(name: [Setting[:default_global_registration_item], 'Global Registration'])
+    templates.update_all(template_kind_id: template_kind.id)
```

Comment:
> gobal template to all OSes which don't have any assigned?

Do you mean registration template? Because assigning global registration template is exactly what we don't want to do.

Reply:
sorry, of course I meant HRT (Linux registration template). The thing this PR autoassignes to newly created hosts :-)